File: app/src/main/java/dmitriy/deomin/generateawallpaper/Adapter_shem.java
package dmitriy.deomin.generateawallpaper;

import android.content.Context;
import android.widget.SimpleAdapter;

import java.util.List;
import java.util.Map;

/**
 * Created by dimon on 15.01.17.
 */

public class Adapter_shem extends SimpleAdapter {
    public Adapter_shem(Context context, List<? extends Map<String, ?>> data, int resource, String[] from, int[] to) {
        super(context, data, resource, from, to);
    }
}

File: app/src/main/java/dmitriy/deomin/generateawallpaper/Adapter_shem.kt
package dmitriy.deomin.generateawallpaper

import android.content.Context
import android.widget.SimpleAdapter

/**
 * Created by dimon on 15.01.17.
 */

class Adapter_shem(context: Context, data: List<Map<String, *>>, resource: Int, from: Array<String>, to: IntArray) : SimpleAdapter(context, data, resource, from, to)
